api/email.py

```diff
@@ -33,11 +34,11 @@ def post(self, request):
         Creates a new feedback email and sends it to admins
         """
         data = request.data
-        required = ['message',]
+        required = ['message', ]
```

Comment:
No problem leaving it, I was  mostly just wondering if there was a reason for it.